core/amber/src/main/scala/edu/uci/ics/texera/workflow/common/WorkflowContext.scala

```diff
@@ -2,14 +2,19 @@ package edu.uci.ics.texera.workflow.common
 import edu.uci.ics.amber.engine.common.virtualidentity.{ExecutionIdentity, WorkflowIdentity}
 import edu.uci.ics.texera.workflow.common.WorkflowContext.{
   DEFAULT_EXECUTION_ID,
-  DEFAULT_WORKFLOW_ID
+  DEFAULT_WORKFLOW_ID,
+  DEFAULT_WORKFLOW_SETTINGS
 }
 
+import edu.uci.ics.texera.workflow.common.workflow.WorkflowSettings
+
 object WorkflowContext {
   val DEFAULT_EXECUTION_ID: ExecutionIdentity = ExecutionIdentity(1L)
   val DEFAULT_WORKFLOW_ID: WorkflowIdentity = WorkflowIdentity(1L)
+  val DEFAULT_WORKFLOW_SETTINGS: WorkflowSettings = WorkflowSettings(400)
```

Comment:
Change "400" to "DEFAULT_BATCH_SIZE"?

Reply:
changed 400 to AmberConfig.defaultBatchSize.